### PR TITLE
Some more cargo orders plus sliceable cabbages

### PR DESCRIPTION
**Pull request by SomeCabbage**
# Description
Adds in a couple of new orders you can make at the cargo request terminal. Also make cabbages sliceable.
Cabbage leaves are obtainable from cutting cabbages. The leaves are unusable in cooking though, so they're only good for raw eating.

- Exotic meats crate: $2500
contains a random assortment and quantity of some more exotic meats to cook and use. Requires kitchen access to open.

- Fox crate: $1200
contains two foxes.

- Cabbage crate: $425
contains 12 cabbages. Bonus chance for one more!

- Science materials crate: $5000
contains a stack of plasma, plasteel and plastic and 1-5 bluespace crystals. Requires science access to open.

- APE crate: $1500
contains an APE unit and an anomaly scanner. Intended to be bought if the starting APE has been destroyed and anomaly tech is not researched or is impossible to get. Requires science access to open.

# Media

# Changelog

:cl: SomeCabbage
- add: Added a few new cargo orders to the request terminal. Includes the Fox crate, Exotic meats, and more.
- add: Cabbages can now be cut into leaves (request by Pspritechologist)

### CODE REVIEW

**Comment by SomeCabbage** [comment]
hold on the crate isnt actually getting kitchen access for some reason

**Comment by SomeCabbage** [comment]
fixed it by making a dedicated secure freezer

File: Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/npc.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+- type: entity
+  id: CrateNPCFox
+  parent: CrateLivestock
+  name: Fox crate
+  description: "A crate containing a single fox."
+  components:
+  - type: StorageFill
+    contents:
+      - id: MobFox
+        amount: 1
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/npc.yml]: just one?

File: Resources/Prototypes/SimpleStation14/Catalog/Cargo/cargo_science.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+- type: cargoProduct
+  id: ScienceMaterials
+  name: Science materials crate
+  description: "A crate containing a varienty of raw materials relevant to research and development"
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Catalog/Cargo/cargo_science.yml]:
```suggestion
  description: "A crate containing a variety of raw materials relevant to research and development"
```

File: Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    - id: FoodMeatPenguin
+      amount: 4
+    - id: FoodMeatRat
+      amount: 4
+    - id: FoodMeatLizard
+      amount: 2
+    - id: FoodMeatSpider
+      amount: 2
+    - id: FoodMeatXeno
+      prob: 0.01
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml]: pretty rare

File: Resources/Textures/SimpleStation14/Objects/Specific/Hydroponics/cabbage.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/SimpleStation14/Objects/Specific/Hydroponics/cabbage.rsi/meta.json]: copyright

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

File: Resources/Textures/SimpleStation14/Objects/Specific/Hydroponics/cabbage.rsi/meta.json (hunk 4 above)
Comment (SomeCabbage) [reply Resources/Textures/SimpleStation14/Objects/Specific/Hydroponics/cabbage.rsi/meta.json]: oops

File: Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml (hunk 3 above)
Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml]: parky's suggestion

File: Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/npc.yml (hunk 1 above)
Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/npc.yml]: same principle as the corgi and cat crate. One pet for 1k

Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/npc.yml]: buuut I can make it more

**Review by DEATHB4DEFEAT — APPROVED** [review]

File: Resources/Prototypes/SimpleStation14/Catalog/Cargo/cargo_science.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    state: base
+  product: CrateScienceMaterials
+  cost: 6500
+  category: Science
+  group: market
+
+- type: cargoProduct
+  id: ScienceAPE
+  name: APE crate
+  description: "A crate containing a single APE unit for containing and poking anomalies"
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Catalog/Cargo/cargo_science.yml]: Poking?

File: Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml (hunk 3 above)
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml]: I would make far more of these fall under a probability, so there's not quite as much meat per crate, but more variety.

File: Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/science.yml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+      - id: SheetPlasma
+      - id: SheetPlastic
+      - id: SheetPlasteel
+      - id: MaterialBluespace
+
+- type: entity
+  id: CrateScienceAPE
+  parent: CrateScienceSecure
+  name: APE crate
+  description: "A crate containing a single APE unit"
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/science.yml]: Clarify here (and in the purchase description) that it has a scanner. Good idea to include it.

File: Resources/Prototypes/SimpleStation14/Entities/Structures/Storage/Crates/crates.yml [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  parent: CrateBaseSecure
+  id: CrateFreezerSecure
+  name: secure freezer
+  components:
+    - type: Icon
+      sprite: SimpleStation14/Structures/Storage/Crates/freezer_secure.rsi
+    - type: Sprite
+      sprite: SimpleStation14/Structures/Storage/Crates/freezer_secure.rsi
+    - type: AccessReader
+      access: [["Kitchen"]]
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Structures/Storage/Crates/crates.yml:11]: I am unsure of how wise it is to give Chefs alone access to several poisonous meats. Chef already has access to a _lot_ of murder tools.

**Review by Pspritechologist — CHANGES_REQUESTED** [review]
I assume you've double checked the price of all the crates? Price of a fox, an APE, etc.

File: Resources/Prototypes/SimpleStation14/Entities/Structures/Storage/Crates/crates.yml (hunk 7 above)
Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Entities/Structures/Storage/Crates/crates.yml:11]: well uh what access should it go to then?

File: Resources/Prototypes/SimpleStation14/Catalog/Cargo/cargo_science.yml (hunk 5 above)
Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Catalog/Cargo/cargo_science.yml]: poking/prodding with particles for research

File: Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml (hunk 3 above)
Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml]: Hm is it possible to make them count towards a total amount? Or perhaps give it a one or the other kinda chance?

Comment (Pspritechologist) [reply Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml]: Don't believe so. See maints lockers, for instance.

File: Resources/Prototypes/SimpleStation14/Entities/Structures/Storage/Crates/crates.yml (hunk 7 above)
Comment (Pspritechologist) [reply Resources/Prototypes/SimpleStation14/Entities/Structures/Storage/Crates/crates.yml:11]: Security was my thought initially. I'm not entirely against this, just uncertain.

Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Entities/Structures/Storage/Crates/crates.yml:11]: oh I didnt check the price

**Comment by SomeCabbage** [comment]
oh shoot I didnt check the prices

**Comment by DEATHB4DEFEAT** [comment]
> oh shoot I didnt check the prices

if only tests worked..

**Comment by SomeCabbage** [comment]
well I fixed them so they are more accurate to their in-game value and also not hyperinflated too and reasonable

**Comment by SomeCabbage** [comment]
changed the values again to be even more appropriate. The meat crate will probably stay at 2.5k due to it being a gamble

**Review by Pspritechologist — COMMENTED** [review]
Does the crate currently have a single bluespace?
Give it a chance to have one to five, the odds getting exponentially lower with each one. It's ok if it ends up being worth more to sell if there happens to be like four of it, maybe three at most.

**Comment by SomeCabbage** [comment]
that actually sounds like a decent idea

File: Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml [hunk 8]
```diff
@@ -0,0 +1,78 @@
+- type: entity
+  id: CrateFoodExoticMeats
+  name: Exotic meats crate
+  description: "Contains a random variety of uncooked, exotic meats. Eat at your own peril."
+  parent: CrateFreezerSecure
+  components:
+  - type: StorageFill
+    contents:
+    # guarenteed meats
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml]:
```suggestion
    # guaranteed meats
```

File: Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+      prob: 0.30
+      amount: 1
+    - id: FoodMeatXeno
+      prob: 0.02
+      amount: 2
+    - id: FoodMeatXeno
+      prob: 0.05
+      amount: 1
+    - id: SheetPlasma1
+      prob: 0.01
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml]: what

**Review by Pspritechologist — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml (hunk 9 above)
Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml]: mmm plasma

Comment (SomeCabbage) [reply Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml]: yeah I'll get rid of it

**Review by Pspritechologist — APPROVED** [review]

**Pull request merged**